apps/web/src/pages/templates/components/email-editor/EmailMessagesCards.tsx

```diff
@@ -131,6 +133,17 @@ export function EmailMessagesCards({ index, isIntegrationActive }: { index: numb
               index={index}
               isIntegrationActive={isIntegrationActive}
             />
+            <ProductLead
+              icon={<Translate />}
+              id="translate-email-editor"
+              title="Translation management"
+              // eslint-disable-next-line max-len
+              text="Translate your notification content to multiple languages using a connection with a preferred i18n localization provider."
```

Comment:
Maybe we can extract this translate strings, so if needs to change it will be in one place.